examples/http_crawl.cpp

```diff
@@ -31,9 +32,10 @@ int main(int, char const*[])
             ip::tcp::resolver r(ios);
             auto it = r.resolve(
                 ip::tcp::resolver::query{host, "http"});
-            stream<ip::tcp::socket> hs(ios);
-            connect(hs.lowest_layer(), it);
-            auto ep = hs.lowest_layer().remote_endpoint();
+            streambuf sb;
```

Comment:
The tiniest of nits: I'd move the `sb` declaration down close to the first use on line 49.  Your call.